package/MDAnalysis/topology/guessers.py

```diff
@@ -202,9 +202,14 @@ def guess_atom_element(atomname):
     try:
         return tables.atomelements[atomname.upper()]
     except KeyError:
-        # strip symbols and numbers
+        # strip symbols
         no_symbols = re.sub(SYMBOLS, '', atomname)
-        name = re.sub(NUMBERS, '', no_symbols).upper()
+
+        # split name by numbers
+        no_numbers = re.split(NUMBERS, no_symbols)
+        no_numbers = list(filter(None, no_numbers)) #remove ''
+        # if no_numbers is not empty, use the first element of no_numbers
+        name = list(filter(None, no_numbers))[0].upper() if no_numbers else ''
```

Comment:
Apologies if I'm missing something obvious, but do you need to apply filter again here? You've already removed the `''` entries and you shouldn't have any `None` entries ?

Reply:
Sorry, I think I made a mistake while modifying the code...